[
    {
        "title": "fix image upscale on cpu"
    },
    {
        "author": {
            "login": "PR Description"
        },
        "body": "## Description\r\n@AUTOMATIC1111 @akx \r\n\r\n- after https://github.com/AUTOMATIC1111/stable-diffusion-webui/pull/16144\r\nfor some reason upscale using cpu will fail with\r\nRuntimeError: Inplace update to inference tensor outside InferenceMode\r\nswitch from `no_grad` to `inference_mode` seems to have fixed it\r\n\r\nI've mentioned this on discord and ask if anyone is able to reproduce this issue but no one give me a proper answer so I assume that it was something wrong with my system \r\n> when I was testing backthen I think this error only happens when using `torch+cuda` with `--use-cpu all` arg\r\n> and it some how works in torch (cpu only)\r\n> but when I retested just now both CPU and CUDA version of torch failed\r\n> so I'm not entirely sure what's going on\r\n\r\nnow that someone has also reported on the same issue so I think it's not just me\r\n- https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues/16274\r\n\r\nas this issue could have quite a large impact for those people that are mainly using webui with CPU to upsacall image and not SD\r\nI suggest a version 1.10.1 patch\r\n\r\n## Checklist:\r\n\r\n- [x] I have read [contributing wiki page](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing)\r\n- [x] I have performed a self-review of my own code\r\n- [x] My code follows the [style guidelines](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Contributing#code-style)\r\n- [x] My code passes [tests](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Tests)\r\n"
    },
    {
        "author": {
            "login": "AUTOMATIC1111"
        },
        "body": ""
    }
]